cmd/bee/cmd/start.go

```diff
@@ -254,7 +254,7 @@ func waitForClef(logger logging.Logger, maxRetries uint64, endpoint string) (ext
 			return nil, err
 		}
 		maxRetries--
-		logger.Errorf("cannot connect to clef signer: %v", err)
+		logger.Warningf("failing to connect to clef signer: %v", err)
```

Comment:
I think the wording on the left is better

Reply:
@acud "on the left"? you mean the original? my problem with the original is that it doesn't communicate that it's an ongoing process, and that the failure is temporary. hence my issue with the use of `Errorf`, and the original wording that suggests that it's something final.

note that this change was motivated by an actual confusion of mine, that lasted until i looked at the code.

Reply:
an alternative: `trying to connect to clef signer, but failing due to error %v`

Reply:
This is not really in the English standard-library :) trying and failing at the same time? failed since the failure is already known therefore in past tense. Sorry for the grammar harassment.  

> "on the left"? you mean the original

yes

Reply:
the grammar of spoken languages is not set in stone... ;) i don't care how, but what i pointed out should be expressed in a way that is not confusing. the original wording suggests a final, failed situation.

i still think it's clear english: the trying is still happening, continuously, but it also keeps failing, for now. but it's rather funny when two non-native speakers get into such discussions... :)

Reply:
right. let's ask a native then. @significance?

Reply:
No strong opinion here. I would like to ask @significance to cut the ties here.

Reply:
i guess maybe @attila-lendvai's rephrased alternative is the most proper? `trying to connect to clef signer, but failing due to error %v` but i think any would work for the use case presuming they are appearing multiple times in the logs?

personal vote goes to @acud's lolzcatz version 😹

Reply:
FTR, it usually appears only once in the log, because at the first retry clef is already ready and accepting connections.